agent/sbd

```diff
@@ -82,7 +82,7 @@ off|reset)
 status)
     sbd_check_device
     sbd_validate_timeout
-    if ! sbd -d $sbd_device list >/dev/null 2>&1 ; then
+    if ! ps aux|grep "sbd: inquisitor"|grep -v grep; then
```

Comment:
I'd not replace the 'list 'check, but additionally check the existence of the sbd, probably the servant processes, would be a plus I think, especially if sbd.service is mistakenly disabled or sbd processes encountered any issues to start/spawn.

And I'd prefer using pgrep. Besides,  check the status of processes with "kill -0" would be even better.

Reply:
Oh, we probably don't even need pgrep since actually there's  sbd.pid file specified in sdb.service.